objects/src/accounts/account_id.rs

```diff
@@ -140,14 +144,58 @@ impl AccountId {
         ];
         let mut current_digest = Self::compute_digest(current_seed, code_root, storage_root);
 
+        #[cfg(feature = "log")]
+        {
+            log::info!(
+                "Generating new account seed [pow={}, current_digest={}, current_seed={}]",
+                digest_pow(current_digest),
+                digest_hex(current_digest),
+                word_hex(current_seed),
+            );
+        }
```

Comment:
Similar comment to above - I don't think extra curly braces are needed.

Reply:
removed